jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -292,8 +314,8 @@ boolean isRPCServer() {
         }
       }
 
-      if (parent == null || isRPCServer()) {
-        // add tracer tags only to first span in the process
+      // add tracer tags to zipkin first span in process
+      if (zipkinSharedRPCSpan && (preferredParent == null || isRPCServer())) {
```

Comment:
Originally I had it there, but I had to tweak zipkin thritf convertor and it did not look right, this was cleaner.